chapters/renderpass.txt

```diff
@@ -1609,21 +1609,35 @@ attachments on multiple uses via ename:VK_ATTACHMENT_LOAD_OP_CLEAR.
   * [[VUID-VkAttachmentDescription-finalLayout-00843]]
     pname:finalLayout must: not be ename:VK_IMAGE_LAYOUT_UNDEFINED or
     ename:VK_IMAGE_LAYOUT_PREINITIALIZED
+ifndef::VK_VERSION_1_1,VK_KHR_maintenance2[]
```

Comment:
I'm unclear why you added these #ifndefs. AFAIK the desired end state is that none of these 4 enums are valid layouts for a color format. Because these two enums are always defined while the other two are introduced by maintenance4 / VK_VERSION_1_1, it's appropriate to have ifdefs around the latter two, but not to have ifndefs around the first two.

Reply:
I understand you point here. Your idea is from the perspective of this project itself. But I pay more attention on the html file generated. Because the project itself is just like a tool to generate Vulkan spec in html. The former is used by a very small amount of experts, while the latter is used by much more developers. 

And the ifndef-endif-ifdef-endif structure will make it cleaner for the html generated. No subsequent repetition text like `If format is a color format, initialLayout must not be`. 

And this method (ifndef-endif-ifdef-endif) is applied even in current project itself, for example: 
  - VUID 03062 and 04953, 
  - VUID 00896 and 01758. 

However, I agree that Vulkan and Vulkan spec are very verbose (intentionally?). Repetitions/duplications are here and there in its spec (html file). For example, the `initialLayout` and `finalLayout` in this PR can be combined if they are in other specifications. But Vulkan spec doesn't do this. It even divide them into many scattered UVIDs respectively. So I followed your suggestion and didn't apply ifndef-endif-ifdef-endif block. PTAL. Thank you!